ocfkube/__init__.py

```diff
@@ -32,7 +32,10 @@ def write_manifests(objects, appname: str, manifest_dir: Path):
         name = obj["metadata"].get("name", obj["metadata"].get("generateName", None))
         kind = obj["kind"]
         namespace = obj["metadata"].get("namespace", appname)
-        with open(appdir / f"{name}_{kind}_{namespace}.yaml", "w") as f:
+        manifest_name = f"{name}_{kind}_{namespace}.yaml"
+        # sanitize for Windows paths :sob:
+        manifest_name = manifest_name.replace(':', '-')

```

Comment:
Are we sure this substitution is always safe? I know you can have - in kube object names, so maybe we use _ again? This makes our file names not-machine-parseable but it should work...